install/kubernetes/helm/istio/charts/security/values.yaml

```diff
@@ -17,6 +17,7 @@ podAnnotations: {}
 citadelHealthCheck: false
 # 90*24hour = 2160h
 workloadCertTtl: 2160h
+maxWorkloadCertTtl: 2160h
```

Comment:
Why would a user set this? Without documentation about the differences between `workloadCertTtl` and `maxWorkloadCertTtl` this is very confusing

Reply:
add comments(copied below differences from citadel's code to here); 

https://github.com/istio/istio/blob/master/security/cmd/istio_ca/main.go#L236
```
	flags.DurationVar(&opts.workloadCertTTL, "workload-cert-ttl", cmd.DefaultWorkloadCertTTL,
		"The TTL of issued workload certificates.")
	flags.DurationVar(&opts.maxWorkloadCertTTL, "max-workload-cert-ttl", cmd.DefaultMaxWorkloadCertTTL,
		"The max TTL of issued workload certificates.")
```

Reply:
By looking at the code, this only affects SDS. Do we need to mention that in the flag description? @myidpt

Reply:
I double check this code, file mount secret is also controlled by this maxTTL. This is not fully covered by test, could you add test for this field?

Reply:
Yes, please add test for this.

Reply:
well, does Istio security team have bandwidth to fix this bug ? currently customer cannot extend the cert life time using citadel.